pipeline/measuring.py

```diff
@@ -217,13 +241,13 @@ def run(self, *args, **kwargs):
 
                 # Apply analytic cuts to each stamp image, to rule out artefacts.
                 m.disqualifier_scores = {}
-                if m.background != 0 and m.background_err > 0:
+                if m.background != 0 and m.background_err > 0.1:
```

Comment:
Hmm, OK.  Could we at least put in a warning?  That way, if things are going haywire because it did (somehow) come up in real life, somebody looking at the error stream trying to figure out what happened might have a hope of seeing this warning.